src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/CommandBuilders/ASPNetBenchmarks.CommandBuilder.cs

```diff
@@ -75,7 +75,8 @@ public static (string, string) Build(ASPNetBenchmarksConfiguration configuration
             commandStringBuilder.Append($" --application.framework net8.0 ");
 
             string corerunToSend = run.Value.corerun.EndsWith("\\") ? run.Value.corerun.Remove(run.Value.corerun.Length - 1) : run.Value.corerun;
-            commandStringBuilder.Append($" --application.options.outputFiles {Path.Combine(Path.GetDirectoryName(corerunToSend), "*.*" )}");
+            commandStringBuilder.Append($" --application.options.outputFiles {Path.Combine(Path.GetDirectoryName(corerunToSend), "coreclr.dll" )}");
```

Comment:
even if we hardcoded, we shouldn't copy coreclr.dll, because that could certainly introduce mismatch (with system.private.corelib.dll and/or clrjit.dll). 

it seems to me that if the user specifies a single file, the intention is clear - they want to copy over just that file (if you change it to *.* it means I'd have to create a new dir and just put that file there if I want to avoid copying over the whole dir - I myself almost never want to copy over the whole corerun dir); and if they specify a directory, it means it's desirable to copy *.* from that directory. 

what do you guys think?

Reply:
I agree - we'll just need to make the corresponding changes to the Create Suite functionality for the ASP.NET suite creation to now target the corerun folder rather than uploading _just_ the corerun file.